packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -97,9 +97,9 @@ class SimControl {
     final Map<String, dynamic> devicesSection = _list(SimControlListSection.devices);
 
     for (String deviceCategory in devicesSection.keys) {
-      final List<Map<String, String>> devicesData = devicesSection[deviceCategory];
+      final List<dynamic> devicesData = devicesSection[deviceCategory];
```

Comment:
Perhaps something like:

```
  final List<dynamic> devicesData = devicesSection[deviceCategory];
  for (Map<String, dynamic> data in devicesData.map(castStringKeyedMap)) {
    devices.add(new SimDevice(deviceCategory, data));
  }
```

and `castStringKeyedMap` would look like:

```
Map<String, dynamic> castStringKeyedMap(dynamic untyped) {
  final Map<dynamic, dynamic> map = untyped;
  return map.cast<String, dynamic>();
}
```

The casting will ensure that the items in the list we iterate over are `Map<String, dynamic>` typed, and not `Map<dynamic, dynamic>`.

Reply:
Done.